docs/node-operators/get-started/setup-datanode.md

```diff
@@ -336,18 +351,180 @@ vega datanode start --home="YOUR_DATA_NODE_HOME_PATH"
 ## Fetch network history
 After starting a data node, you can load in a segment of network history, if you want your node to have more data than provided by the current height. This is particularly useful if you're running an archive node.
 
+Before you can fetch network history, you will need to update your data node configuration file to add a list of bootstrap peers, as by default this is left empty and depends on which network you are connecting to:
+
+```toml
+[NetworkHistory]
+...
+  [NetworkHistory.Store]
+  ...
+  BootstrapPeers = ["/path/to/bootstrap-peer/1","/path/to/bootstrap-peer/2"]
+```
+
+To get a list of bootstrap peers available for your network, you can make a HTTP request to the API service for your chosen network, for example: `https://api.testnet.vega.xyz/api/v2/networkhistory` will retrieve the network peers for `testnet`.
+
+```json
+{
+  "ipfsAddress": "/ip4/<data-node-ip-4-address>/tcp/4001/p2p/<some-uid>",
+  "swarmKey": "/key/swarm/psk/1.0.0/\n/base16/\n<some key>",
+  "swarmKeySeed": "<some-key-seed>",
+  "connectedPeers": [
+    "/ip4/<peer-ip-4-address>/tcp/4001/p2p/<peer-id>",
+    ...
+  ]
+}
+```
+
+
 To see how much network history your data node has, run the following command:
 
 ```shell
 vega datanode network-history show --home="YOUR_DATA_NODE_HOME_PATH"
 ```
 
-To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the oldest) followed by the number of blocks prior to the segment's height that you want fetch. `2000` is used in the following example. This will result in all blocks from height 3000 to 5000 being retrieved.
+### Fetching network history for a new data node
+
+If you are building a new data node, you may have no history and you will see this message:
+
+```shell
+No network history is available. Use the fetch command to fetch network history
+```
+
+In this case, it is possible to get the latest history segment ids from your network history bootstrap peers:
+
+```shell
+vega datanode network-history latest-history-segment-from-peers --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+This will return a list of peers you can fetch network history from:
+
+```text
+Most Recent History Segments:
+
+Peer:<peer-ip-and-api-port>, Swarm Key:<swarm-key>, Segment{from_height:75001  to_height:75300  history_segment_id:"some-segment-id"  previous_history_segment_id:"some-other-segment-id"}
+...
+```
+
+:::note
+It is possible that some data peers are behind and may not have the latest data. However, the `history_segment_id` should be the same across all nodes that are at the same height.
+:::
+
+
+To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the oldest) followed by the number of blocks prior to the segment's height that you want fetch. `2000` is used in the following example. This will result in all blocks from height N-2000 to N being retrieved.
```

Comment:
```suggestion
To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the latest) followed by the number of blocks prior to the segment's height that you want fetch. `2000` is used in the following example. This will result in all blocks from height N-2000 to N being retrieved.
```